lib/pages/chat/events/html_message.dart

```diff
@@ -163,6 +169,20 @@ class HtmlMessage extends StatelessWidget {
         }
         return {};
       },
-    );
+      pillBuilder: (identifier, url, onTap, getMxcUrl) {
+        debugPrint('identifier: $identifier, url: $url');
+        if (event!.status == EventStatus.sending) {
```

Comment:
if `event == null` what is happen?

Reply:
event can't be null, that is for sure, `event` is nullable because other class also use this HtmlMessage and don't need event

Reply:
but still need to keep the thing in safe. respect the input to keep us safe in the future also